doc/configcommands.dsv

```diff
@@ -73,7 +73,8 @@ oldreader-flag-star|<flag>|""|If this is set and The Old Reader support is used,
 oldreader-login|<login>|""|This variable sets your The Old Reader login for The Older Reader support.|oldreader-login "your-login"
 oldreader-min-items|<number>|20|This variable sets the number of articles that are loaded from The Old Reader per feed.|oldreader-min-items 100
 oldreader-password|<password>|""|This variable sets your The Old Reader password for The Old Reader support. Double quotes should be escaped, i.e. you should write +{backslash}"+ instead of +"+.|oldreader-password "here_goesAquote:{backslash}""
-oldreader-passwordfile|<path-to-file|""|A more secure alternative to the above, by storing your password elsewhere in your system.|oldreader-passwordfile "path-to-file"
+oldreader-passwordfile|<path-to-file>|""|A more secure alternative to the above, by storing your password elsewhere in your system.|oldreader-passwordfile "path-to-file"
+oldreader-passwordeval|<command>|""|An even more secure alternative, is storing your password encrypted and decrypting it with an external command. For example with gpg or your systems keyring.|oldreader-passwordeval "command some-parameter"
```

Comment:
Well.. storing the password in another file isn't any "more secure" as well. By storing it in a plain text file you can't get it to be more secure than just setting the config file to mode `0600`.
I know, the reason behind it is do be able to publish your config and keep the passwordfile local, but this doesn't stop the user from unintentionally publishing his passwordfile as well.

But I see your point, will change it to be more "neutral". Someone knowing what to look for should find it anyway.